src/modules/jcms_rest/src/Plugin/rest/resource/PressPackageListRestResource.php

```diff
@@ -87,8 +87,8 @@ public function getItem(EntityInterface $node) {
     // Subjects are optional.
     if ($node->get('field_related_content')->count()) {
       $articles = [];
-      foreach ($node->get('field_related_content') as $related) {
-        if ($article = $this->getArticleSnippet($related->get('entity')->getTarget()->getValue())) {
+      foreach ($node->get('field_related_content')->referencedEntities() as $related) {
```

Comment:
This is just a simplification. It  brings it inline with: https://github.com/elifesciences/journal-cms/blob/develop/src/modules/jcms_rest/src/Plugin/rest/resource/PressPackageItemRestResource.php#L68-L72